src/database/initialization/organization_to_json.sql

```diff
@@ -1,11 +1,31 @@
 CREATE OR REPLACE FUNCTION organization_to_json(organization organizations)
 RETURNS JSONB AS $$
+DECLARE
+  proposal_field_values_json JSONB;
 BEGIN
+  SELECT jsonb_agg(
+    proposal_field_value_to_json(pfv.*)
+    ORDER BY bf.id
+  )
+  INTO proposal_field_values_json
+  FROM proposal_field_values pfv
+  INNER JOIN application_form_fields aff
+    ON pfv.application_form_field_id = aff.id
+  INNER JOIN base_fields bf
+    ON aff.base_field_id = bf.id
+  INNER JOIN proposal_versions pv
+    ON pfv.proposal_version_id = pv.id
+  INNER JOIN organizations_proposals op
+    ON pv.proposal_id = op.proposal_id
+  WHERE op.organization_id = organization.id
+    AND bf.scope = 'organization'
+    AND pfv.is_valid;
```

Comment:
I expect to expand this query further with a bunch of `case` statements etc., but my question for now is whether the gist of this approach is satisfactory, namely a modification to this postgres function and an additional field called `fields`.

Reply:
This seems like the right approach to me yes!